components/buttons/button.css

```diff
@@ -58,14 +59,14 @@
     color: var(--primary-color);
 }
 
-.primary-outlined-btn-md,
-.primary-outlined-btn-xsm,
-.primary-outlined-btn-sm,
-.primary-outlined-btn-lg,
-.label-icon-outlined-btn-md,
-.label-icon-outlined-btn-xsm,
-.label-icon-outlined-btn-sm,
-.label-icon-outlined-btn-lg {
+.primary-outline-btn-md,
+.primary-outline-btn-xsm,
+.primary-outline-btn-sm,
+.primary-outline-btn-lg,
+.label-icon-outline-btn-md,
+.label-icon-outline-btn-xsm,
+.label-icon-outline-btn-sm,
+.label-icon-outline-btn-lg {
```

Comment:
create a class outline and add into it.

Reply:
If I create a class with the name 'outline' then, the user would have to specify one more class to create that component.